doc/onboarding/view/step1.md

```diff
@@ -0,0 +1,59 @@
+
+## Step 1/5 : Welcome!
+
+### URI
+
+`:userID/welcome`
+
+### Models
+
+<pre>
+user = {
+	&lt;string&gt; name,
+	&lt;id&gt; id,
+	&lt;email&gt; email,
+	&lt;boolean&gt; is_registered: false
+}
+</pre>
+<pre>
+step = {
+	&lt;string&gt; slug: 'welcome',
+	&lt;step&gt; value: 1/5
+}
+</pre>
+
+### Getters
+
+<pre>
+    getNextStepURI: (state) ->
```

Comment:
A `getter` is a specific object that doesnt belong to views or models. Views get data from this object that will get data from models so that views never calls models directly (immutability). That's a pattern used with React and Redux framework.

I d'on thing that you implemented such a thing; my question is : should we create one? Where should it be?